torch/csrc/jit/passes/lower_tuples.cpp

```diff
@@ -40,6 +44,17 @@ static void VisitNode(Node* n, Node* insert_point) {
     }
     return;
   }
+
+  // undo TupleUnpack(TupleIndex)
```

Comment:
This comment is wrong. This code is undoing:

`TupleIndex[index=i](TupleConstruct(...))`